components/dropdown/dropdown-content-mixin.js

```diff
@@ -348,7 +396,7 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 			return;
 		}
 
-		this.opened = false;
+		this.close();
```

Comment:
Does this mean that consumers need to call `open()`/`close()` instead of setting `opened = true` or `opened = false`?